mava/wrappers/observation.py

```diff
@@ -17,20 +17,26 @@
 import chex
 import jax.numpy as jnp
 from jumanji import specs
-from jumanji.env import Environment
 from jumanji.types import TimeStep
 from jumanji.wrappers import Wrapper
 
-from mava.types import Observation, ObservationGlobalState, State
+from mava.types import MarlEnv, Observation, ObservationGlobalState, State
 
 
 class AgentIDWrapper(Wrapper):
     """A wrapper to add a one-hot vector as agent IDs to the original observation.
     It can be useful in multi-agent environments where agents require unique identification.
     """
 
-    def __init__(self, env: Environment):
+    # This init isn't really needed as jumanji.Wrapper will forward the attributes,
```

Comment:
Since this already started to improve Mava's generalization can't we also add  a wrapper like MavaEnvWrapper? This might fix these issues once for all for the future and reduce redundancy :thinking:

I believe this will resemble to what jumanji is doing if it's not than it can be dealed with later or ignore it :sweat_face: